recipe/meta.yaml

```
@@ -30,7 +30,12 @@ requirements:
     - {{ compiler('cxx') }}
     - {{ compiler('cuda') }}  # [cuda_compiler_version != "None"]
     # https://conda-forge.org/docs/maintainer/knowledge_base.html?highlight=cuda#using-centos-7
-    - sysroot_linux-64 2.17  # [linux64]
+    - sysroot_linux-64 2.17  # [linux64]    
```

It looks like some extra spaces crept in at the end. So trimmed them

Also this can be generalized to handle all Linux architectures. So rewrote that way below

```suggestion
    - sysroot_{{ target_platform }} 2.17  # [linux]
```